.github/workflows/publish.yml

```diff
@@ -365,22 +365,14 @@ jobs:
       ]
     runs-on: ubuntu-latest
     permissions:
-      contents: write # Grant write access to contents
+      contents: write
+      packages: write
```

Comment:
think these could prolly be safely removed

Reply:
Should be safe according to the docs, yes.